File: art_md_sim_02_1/SingleStep.jl
function SingleStep(mol)
    global stepCount, timeNow
    stepCount = stepCount + 1
    timeNow = stepCount*deltaT
    #
    LeapfrogStep(1,mol)
    ApplyBoundaryCond()
    ComputeForces(mol)
    LeapfrogStep(2,mol)
    #
    EvalProps()
    AccumProps(1)
    #
    if stepCount % stepAvg == 0
        AccumProps(2)
        #PrintSummary()
        AccumProps(0)
    end
end

File: ART_MD/PROG_02_1_C/SingleStep.c
void SingleStep ()
{
  ++ stepCount;
  timeNow = stepCount * deltaT;
  LeapfrogStep (1);
  ApplyBoundaryCond ();
  ComputeForces ();
  LeapfrogStep (2);
  EvalProps ();
  AccumProps (1);
  if (stepCount % stepAvg == 0 || stepCount == 1) {
    AccumProps (2);
    PrintSummary (stdout);
    AccumProps (0);
  }
}
